lifecycle_event.go

```diff
@@ -0,0 +1,24 @@
+package cyclist
+
+import "time"
+
+var (
+	standardFluxCapacitorTime, _ = time.Parse(time.RFC3339, "1955-11-05T11:05:55-09:00")
+)
+
+type lifecycleEvent struct {
+	Event     string    `json:"event"`
+	Timestamp time.Time `json:"timestamp"`
+}
+
+func newLifecycleEvent(event, ts string) *lifecycleEvent {
+	timestamp, err := time.Parse(time.RFC3339Nano, ts)
+	if err != nil {
+		timestamp = standardFluxCapacitorTime
```

Comment:
[Sorry not sorry](https://www.youtube.com/watch?v=OiD4Sm5Vbwk)?

It's effectively `nil`, but also serves the purpose of being a sortable "old" date that's always the same, just like you're saying.